releases/release-1.24/release-notes/maps/pr-106956-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 106956
+releasenote:
+  text: Kubernetes is now built with Golang 1.17.5.
```

Comment:
Need to clarify if this release note should be removed in favor of #107612 which is in the draft also

Reply:
I guess it should be..given #107612 states a higher version being put to use..